src/__tests__/button-test.tsx

```diff
@@ -44,18 +64,13 @@ test('<a> is rendered when using "to" prop', () => {
         </ThemeContextProvider>
     );
 
-    const anchor = screen.getByRole('button', {name: 'test'});
+    const anchor = screen.getByRole('link', {name: 'test'});
```

Comment:
I know this is an expected change, but it will probably break many tests in webapp